schemas/magnetics/magnetics_flux_loop_type_identifier.xml

```diff
@@ -10,5 +10,5 @@ Type of flux loop
 <int name="diamagnetic_internal" description="Diamagnetic internal loop"                   >3</int>
 <int name="diamagnetic_external" description="Diamagnetic external loop"                  >4</int>
 <int name="diamagnetic_compensation" description="Diamagnetic compensation loop" >5</int>
-<int name="diamagnetic_differential"   description="Diamagnetic differential loop"          >6</int>
+<int name="flux_differential"           description="Differential flux loop"                     >6</int>
```

Comment:
Since this will be used for both flux loops and diamagnetic loops, I think we should try to better reflect this in the name and description:
```suggestion
<int name="differential"           description="Difference between loops"                     >6</int>
```